jbmc/regression/jbmc-concurrency/get-current-thread/test3.desc

```diff
@@ -0,0 +1,7 @@
+KNOWNBUG
```

Comment:
As above.

Reply:
done.  test2.desc and test3.desc do not work due to the same bug. I created another  test 'get-current-thread/test_bug.desc' that showcases the issue.   I also added comment over there explaining what the issue is exactly.